boat_interfaces/src/rudder_node.py

```diff
@@ -59,7 +60,7 @@ def compass_callback(compass):
 	wind_heading = (ane_reading + compass.data) % 360
```

Comment:
I feel like angle wrapping should be in the utility class as well

Reply:
Whoops, missed this one. Also found another leftover mod in `boat_nav/src/sensor_parser_node.py` ln 61. Will replace with the correct utility.